plugins/flatpak/gs-flatpak.c

```diff
@@ -442,6 +442,11 @@ gs_flatpak_mark_apps_from_usb_remote (GsFlatpak *self,
 				 app_collection_id, ref_collection_id);
 			continue;
 		}
+
+		// TODO: to the USB category (possibly not in here), add top
+		// banner with "copy OS to USB" button (which calls
+		// eos-updater-prepare-flatpak upon click)
```

Comment:
Use C-style comments.

Reply:
Fixed locally (though this will disappear when implemented)